agent/util-scripts/require-rpm

```diff
@@ -1,20 +1,65 @@
 #!/bin/bash
 # -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 4; sh-indentation: 4; tab-width: 8 -*-
 
+# require-rpm - check to see if a RPM with the required version is installed.
+#
+# Returns 0 (success) if it is; 1 if it isn't; 2 if an error occured during
+# operation.
+
 _rpm="${1}"
 if [[ ! -z "${2}" ]]; then
     _version="-${2}"
 else
     _version=""
 fi
-if [[ ! -z "${_PBENCH_UNIT_TESTS}" ]]; then
-    # Package changes are always successful for unit tests.
-    exit 0
+if [[ ! -z "${3}" ]]; then
+    _match="${3}"
+else
+    _match="eq"
 fi
 
-_status=$(rpm --query ${_rpm}${_version})
+_status=$(rpm --query --queryformat="%{EVR}" ${_rpm} 2>/dev/null)
```

Comment:
Maybe add a comment about what  `EVR` means. Is it `<epoch>-<version>-<release>` with intelligent reformatting if something is missing?

`_status` seems like a wrong name for this variable: maybe `_evr` if the acronym is already explained.